frontend/src/components/DecayInformation.vue

```diff
@@ -62,6 +62,38 @@
               </span>
             </div>
           </div>
+          <div v-if="decay.balance > 0">
+            <div class="d-flex">
+              <div style="width: 40%" class="text-right pr-3 mr-2">
+                <div v-if="type === 'send'">{{ $t('form.amount') }}</div>
```

Comment:
Why do we make a difference between send and receive since it will always be $t('form.amount')?
Do we need it later?